xclim/ensembles/_robustness.py

```diff
@@ -152,12 +164,16 @@ def change_significance(
         p_change = kwargs.setdefault("p_change", 0.05)
 
         # Test hypothesis of no significant change
+        input_core_dims = [[realization, "time"], [realization]]
+        input_core_dims = remove_emptys_from_list(input_core_dims)
+        output_core_dims = [[realization]]
+        output_core_dims = remove_emptys_from_list(output_core_dims)
```

Comment:
I think the code would be easier to read with something like
```
if realization is None:
  input_core_dims=[['time'], []]
  output_core_dims=[[]]
else:
  ...
```

Reply:
or I wonder if it would make sense to create a dummy `realization` dimension in case it doesn't exist, so the internal code doesn't have to handle this special case.